app/views/profiles/show.html.erb

```diff
@@ -27,7 +27,11 @@
     <aside class="px-5">
       <section class="text-center d-flex flex-column align-items-center">
         <div class="ratio ratio-1x1 rounded-circle overflow-hidden w-25">
-          <%= image_tag @profile.photo_attachment, alt: 'Foto de perfil', class: 'img-cover' if @profile.photo.present? %>
+          <% if @profile.photo.present? && @profile.valid? %>
+            <%= image_tag @profile.photo, alt: 'Foto de perfil', class: 'img-cover' %>
+          <% else %>
+            <%= image_tag 'default_portfoliorrr_photo.png', alt: 'Foto de perfil', class: 'img-cover' %>
+          <% end %>
```

Comment:
Isso aqui também está em [app/views/profiles/edit.html.erb](https://github.com/TreinaDev/td11-portfoliorrr/pull/131/files#diff-6091542e4570d68ca546a4a893b0d0202a75b9aa6fabf4e8aa0b4122ae6a3544), como é repetido só uma vez acho que não tem problema, mas se for usar de novo no futuro pode ser interessante a gente extrair essa condicional para uma partial

Reply:
Aqui é só uma sugestão pra possível refatoração futura, acho que não precisa alterar no momento

Reply:
Boa meu querido, vou subir um commit com isso já alterando, pra já deixar o código DRY